Automattic/wp-calypso
# State: Cleanup unused site features cleaning and refreshing

#### Changes proposed in this Pull Request

Site features clearing with the `clearSiteFeatures()` action creator and refreshing with `refreshSiteFeatures()` are not used at all. They were never used since they were introduced in #53126.

This PR removes them, together with the related reducer.

#### Testing instructions

Verify Calypso builds and all tests pass: `yarn run test-client client/state/sites/features`
